src/services/ContactsManager.service.js

```diff
@@ -3,47 +3,61 @@ import axios from 'axios';
 
 export default class ContactsManager {
   _identities = new BehaviorSubject([]);
+  _default_sender = new BehaviorSubject(undefined);
 
   identities$ = this._identities.asObservable();
+  default_sender$ = this._default_sender.asObservable();
 
-  async populateContacts() {
-    let results = await axios
+  populateContacts() {
+    return axios
       .get(`http://localhost:1234/api/identities/`, {
         headers: {
           'Content-Type': 'application/json'
         }
       })
-      .then(function (response) {
-        console.log(response.data.results);
-        return response.data.results;
+      .then((response) => {
+        const r = response?.data?.results;
+        if (r) {
+          this._identities.next([...r]);
+        }
+        return r;
       })
-      .catch(function (error) {
+      .catch((error) => {
         console.error(error);
```

Comment:
Do we need to worry about log messages before submitting?

Reply:
Meaning, do we need to clean those up? I'd say if it's an error, no, if it's some random debugging (like if we were checking why a variable was randomly undefined) then we can clean that up for sure.

Reply:
I am not worried about log messages for M1 submission. But in general, we do need to get into the habit of removing them. Or at least, removing the logs that occur globally and repeatedly.